### ISSUE TITLE
bug: 3D view: in follow mode, world wraps visually even for non-wrapping worlds

### ISSUE THREAD

**Issue by SethTisue** [NetLogo/NetLogo#346]
You can see the bug by running Perspective Example, pressing the `follow one-of turtles` button, and opening the 3D view. You'll see things like this:
![screen shot 2013-05-14 at 2 26 41 pm](https://f.cloud.github.com/assets/161079/503118/f5eea43a-bcc3-11e2-953b-3dcd147f493f.png)
Seeing this would make sense in a wrapping world, but it doesn't make sense in a bounded world.

(reported by Uri Wilensky)

**Comment by ERSUCC** [comment]
This is documented and is therefore not technically a bug, although certainly a valid enhancement request. See the 3D docs at [this](https://ccl.northwestern.edu/netlogo/docs/3d.html#:~:text=Topologies%20are%20not%20yet%20supported%20in%20NetLogo%203D%2C%20so%20the%20world%20always%20wraps%20in%20all%20dimensions.) link.

**Comment by SethTisue** [comment]
I meant the ticket to be about regular NetLogo's 3D view, not about NetLogo 3D.

**Comment by ERSUCC** [comment]
Yeah I know, but it's the same code, so the same logic applies. I suppose there are two possible enhancements, then. Either the 3D view should allow wrapping, or there should be separate code running the 3D view in NetLogo 2D and the 3D view in NetLogo 3D.

**Comment by SethTisue** [comment]
I see.

I would _hope_ that one additional boolean flag, disabling the wrapping logic, would suffice without much extra logic being needed... but admittedly I don't recall the code and this view might be too rosy.
